src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -647,6 +713,15 @@ li {
   }
 }
 
+.toggle-business-entry {
+  cursor: pointer;
```

Comment:
You may have a base class for this already (for next time).

Reply:
Will 👀